.travis.yml

```diff
@@ -26,6 +26,10 @@ matrix:
       env: SYMFONY_VERSION='~4.0.0'
     - php: '7.2'
       env: SYMFONY_VERSION='~4.1.0@dev'
```

Comment:
I think we can remove both the 4.1.0@dev and 4.0.0 one. The 4.4.0 one should be enough